Assets/Scripts/Augment/Augment.cs

```diff
@@ -59,4 +63,16 @@ public static void DisableInstance(GameObject instance, AugmentType type)
                 break;
         }
     }
+
+    public static Transform Midpoint(GameObject instance, AugmentType type)
```

Comment:
Hmm, could be that bodies should inherit from augment instead of this static check, but we can postpone that refactor for later

Reply:
Agreed & yeah I ain't gonna do that now haha